### ISSUE TITLE
React dependency outdated

### ISSUE THREAD

**Issue by geekyme** [yahoo/fluxible-immutable-utils#31]
This library breaks on react 0.14.0

```
npm ERR! peerinvalid Peer fluxible-immutable-utils@0.4.0 wants react@^0.13
```

**Comment by dmhood** [comment]
Thanks, will be fixed with https://github.com/yahoo/fluxible-immutable-utils/pull/33

**Comment by dmhood** [comment]
Re-opening since we are having some issues with our tests, will update before too long.

**Comment by dangw** [comment]
any idea when this will be fixed?

**Comment by akshayp** [comment]
@dangw @geekyme Published 0.5.0 which will give you React 14 compatibility

**Comment by dangw** [comment]
thanks!  confirmed it's working.
